test/UtMapTest.hpp

```diff
@@ -9,577 +9,570 @@
 
```

Comment:
Actually I have no idea if you'll want to include variant & iostream here, because we also don't seem to include string and stuff

Reply:
I think I want to turn these into .cpp files and clean-up the includes, making them all header files has basically polluted each one with the ones included above it